pull_request_template.md

```diff
@@ -1,7 +1,3 @@
-<!-- 
-This PR template serves as a guide and reminder for contributors to ensure that the "blog" label is applied appropriately.
```

Comment:
Sorry, I didn't communicate well. This particular comment has two sentences saying the same thing. We should keep  "Please make sure to apply the "blog" label to the pull request if it is related to a blog post so that it can be published to Slack." and remove "This PR template serves as a guide and reminder for contributors to ensure that the "blog" label is applied appropriately."

Reply:
Oh, okay. 
Done.